src/main/java/de/torui/coflsky/Config.java

```diff
@@ -1,5 +1,5 @@
 package de.torui.coflsky;
 
 public class Config {
-    public static final int KeepFlipsForSeconds = 50;
+    public static final int KeepFlipsForSeconds = 42;
```

Comment:
may i ask why this specific change?

Reply:
New auctions for flips are available every 60 seconds and it commonly takes more than 5 seconds to process them.
However People wanted to hold the key down earlier than 5 seconds causing old flips to be opened